drivers/flash/flash_mcux_flexspi_hyperflash.c

```diff
@@ -424,6 +420,11 @@ static int flash_flexspi_hyperflash_write(const struct device *dev, off_t offset
 		return -EINVAL;
 	}
 
+#ifdef CONFIG_HAS_MCUX_CACHE
+	L1CACHE_DisableICache();
+	L1CACHE_DisableDCache();
```

Comment:
Shouldn't you use the `sys_cache_` functions instead? In case the `CACHE_MANAGEMENT` is turned off completely?

Reply:
Yeah, probably best to get this conversion done. Changed to use `sys_cache*` functions in latest push